.eslintrc.yml

```diff
@@ -6,6 +6,10 @@ plugins:
   - react
   - json
 
+settings:
+  react:
+    version: '16.3'
```

Comment:
This is a new feature of `eslint-plugin-react` and it's autodetection doesn't work because we use `app/package.json` to install react, so this needs to be hard-coded for the moment.